string mission_id
string[] task_labels	# list of tasks enabled for executing